windows/windows_update_install/install_msu_file.yml

```diff
@@ -1,10 +1,25 @@
 # Copyright 2023-2024 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 ---
+# Parameters:
+#   msu_file_path (required): the .msu file path in guest OS.
+#   msu_file_basename (optional): the base name of the .msu file.
+#
+- name: "Check if the input parameter is set"
+  ansible.builtin.assert:
+    that:
+      - msu_file_path is defined
+      - msu_file_path
+    fail_msg: "Parameter 'msu_file_path' is required to be set."
+
+- name: "Set the .msu file basename"
```

Comment:
This is not a utilities task, do not need to pass parameters and in below task calling this also do not pass parameters.
`msu_file_path` and `msu_file_basename` are set at the same time, why here set it again?

Reply:
Updated.